http/templates/season_history.html.tera

```diff
@@ -10,72 +10,92 @@
     </div>
     <div class="page-content-container">
         <div class="previous-season-item">
-            <h3 class="season-title">Season 9</h3>
-            Another season of Any% NMG, this time with
-            {{macros::link(href="/season/9", text="four swiss brackets and one round robin bracket")}}.
+            <h3 class="season-title">Season 10: Any% NMG</h3>
+            This time we were back to exactly
+            {{macros::link(href="/season/10/standings", text="three swiss brackets")}}.
```

Comment:
i swapped all these to Standings links because i think it's a better landing page